gm4_shiny_items/data/gm4_shiny_items/functions/make_item_shiny.mcfunction

```diff
@@ -0,0 +1,16 @@
+# Run from as_gm4_can_become_shiny
+# @s = All items with a score of "gm4_absorbed_xp" that is equal or greater to the value needed for an item to become "Shiny"
+
+data remove entity @s Item.tag.unstackable

```

Comment:
What is this tag used for? If this is a custom gm4 tag it should also have the `gm4_` prefix.